packages/graphql/src/mutations/_txHelper.js

```diff
@@ -557,7 +557,18 @@ async function sendViaWeb3({
 
   tx.once('transactionHash', async hash => {
     txHash = hash
-    if (typeof txHash !== 'string' || ![66, 64].includes(txHash.length)) {
+    if (
+      typeof txHash === 'object' &&
+      Object.prototype.hasOwnProperty.call(txHash, 'message')
```

Comment:
could we say `if (get(txHash, 'message'))`? Object.prototype.hasOwnProperty.call seems a bit verbose?

Reply:
Curious about performance(bool check vs string comparison).  Probably trivial, will change it for clarity.